rainbow-me/rainbow
# Custom app icon for SMOL/Treasure

Fixes RNBW-4288, RNBW-4289
Figma link (if any):
https://www.figma.com/file/MbOQyKuRkviDny0kpYDo6j/SMOL-Icon-Explainer-Sheet?node-id=0%3A1

## What changed (plus any additional context for devs)
1. added support for SMOL app icon
2. refactor of existing architecture/logic

## Screen recordings / screenshots
See slack thread for screenshots/recordings

https://rainbowhaus.slack.com/archives/C03QH9C9S7K/p1661294082793969?thread_ts=1661213024.733959&cid=C03QH9C9S7K

## What to test
- TF 1.7.3 (5)
- build on **physical** ios device
- check that the unlockable icons are unlocked no matter what wallet is currently focused. the check should scan all imported wallets at once
- if multiple icons are unlockable, make sure that they are all unlocked at once, but only 1 explain sheet is shown at a time. if there are more explain sheets to be shown, show it on the next app launch.
- test/double check ALL smolverse NFTs. The wallet i was using for testing (0x9Ae4A4C256146D00dd430eAA1413ED43B4fFfc67) only has 1 smol nft


## Final checklist
